nipype/utils/filemanip.py

```diff
@@ -28,6 +28,9 @@
 
 related_filetype_sets = [(".hdr", ".img", ".mat"), (".nii", ".mat"), (".BRIK", ".HEAD")]
 
+# Keep anybody who depended on the old hack from breaking
+which = shutil.which
```

Comment:
just a note, the keyword args (which can also be positional) are different

Reply:
Yeah. I guess we should keep and deprecate the old one instead.

Reply:
Can't you wrap calls to `shutil.which` in `n/u/filemanip.py#which` and keep the old function signature alive ?

I'd expect users who want to use `shutil.which` to import it directly instead of using nipype as a proxy for it anyway. However, anyone using nipype's `which` with named arguments would get an error which is sad. It depends how popular you think these utilities are in client code. I know Clinica uses some and we are trying to move away from them to reduce coupling. 

Or a solution in between would be start with a `DeprecationWarning` advising to port to `shutil.which` and later remove or proxy it?

Reply:
Can't you wrap calls to `shutil.which` in `n/u/filemanip.py#which` and keep the old function signature alive ?

I'd expect users who want to use `shutil.which` to import it directly instead of using nipype as a proxy for it anyway. However, anyone using nipype's `which` with named arguments would get an error which is sad. It depends how popular you think these utilities are in client code. I know Clinica uses some and we are trying to move away from them to reduce coupling. 

Or a solution in between would be start with a `DeprecationWarning` advising to port to `shutil.which` and later remove or proxy it?